ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml

```diff
@@ -56,6 +56,10 @@ tests:
   container:
     from: src
   run_if_changed: \.go$
+- as: verify-generate
+  commands: make verify-generate VERSION=$(grep -m 1 VERSION Makefile | sed 's/^.*= //g')
```

Comment:
This is ugly I notice too.
I tried `make verify-generate`, expecting the version is taken from the default value defined in the Makefile. However, the value would be `1.20`.

https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/54417/rehearse-54417-pull-ci-openshift-cincinnati-operator-master-verify-generate/1816892227054473216#1:build-log.txt%3A51

```console
podman run -it --rm registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.15
WARNING: image platform (linux/amd64) does not match the expected platform (linux/arm64)
[root@02c6f2b07fcc origin]# echo $VERSION
1.20
```

So I need to get the value in the Makefile at the runtime in CI.

Reply:
Can we maybe rename the variable in the Makefile to something like `OPERATOR_VERSION`? It's only used in the  Makefile, Dockerfile and mentioned in the README:

```console
$ git grep -E '[^_]VERSION' -- ':!vendor'
Dockerfile:RUN make GOBUILDFLAGS=-mod=vendor VERSION="$(git describe --abbrev=8 --dirty --always)" build
Makefile:GOLDFLAGS ?= -s -w -X github.com/openshift/cincinnati-operator/version.Operator=$(VERSION)
Makefile:BUNDLE_GEN_FLAGS ?= -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
Makefile:       $(MAKE) bundle VERSION=$(VERSION)
README.md:* Set the `VERSION` value in the shell
docs/disconnected-updateservice-operator.md:> VERSION IMAGE
```

That would prevent us from inheriting the unrelated environmental variable.

Reply:
$VERSION comes from operator SDK https://github.com/operator-framework/operator-sdk/blob/e95abdbd5ccb7ca0fd586e0c6f578e491b0a025b/testdata/go/v4/memcached-operator/Makefile#L6

It might be used by sdk in other places in the future.

I thought about the renaming too but did not go that way as the trouble ("inheriting the unrelated environmental variable") is caused by our CI setup. So I think the solution should be in CI too (e.g., tests defined in the ci-op's config).

I would like to keep this one if you allow for it.
If it causes other issues, I will do the renaming.